docs/README.md

```diff
@@ -1,8 +1,24 @@
-To generate HTML documentation run the `generate_docs.py` script from the `build` dir.
-It will create extra `./docs_build` directory, where the intermediate and final files
+# Documentation
+
+To generate HTML documentation run the `generate_docs.py` script from any sub-dir of the
+repository (most likely `build`) or enable and use build target 'docs' (see details below).
+
+This script will create `./docs_build` sub-directory, where the intermediate and final files
 will be created. HTML docs will be in the `./docs_build/generated/html` directory.
 
-The script requires:
+## make docs
+
+To run documentation generation via build target use CMake commands below.
+To enable this target, python executable has to be found in the system.
```

Comment:
with a minimum version of...

Reply:
done (don't want to duplicate the version, it's defiend below)